src/components/RoleLink/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { useAppSelector } from "@store";
+import { ADMIN, MANAGER, useTypedTranslation, DropDown } from "@shared";
+import { Link } from "react-router-dom";
+
+export default function RoleLink (): JSX.Element {
```

Comment:
RoleLink is not good name of component. What it does? It renders list of navigation links based on the role of current user.
So, please, rename it, thanks.